docs/.vuepress/config.js

```diff
@@ -300,7 +300,7 @@ module.exports = {
             {
               title: "Overview",
               directory: false,
-              path: "/light-clients/overview.html",
+              path: "/ibc/light-clients/overview.html",
```

Comment:
Need to fix other pages too, 404's were being returned when rendering docs with `make view-docs`